lib/engine/graph.rb

```diff
@@ -23,7 +24,7 @@ def clear
       @connected_hexes.clear
       @connected_nodes.clear
       @connected_paths.clear
-      @reachable_hexes.clear
```

Comment:
why did you delete this

Reply:
can you undo these changes and just use a new graph like 1873?

Reply:
I've undid the changes and made a separate graph for this case